main.go

```diff
@@ -304,4 +306,16 @@ func setupServiceOperatorDefaults() {
 	}
 
 	redisv1.SetupRedisDefaults(redisDefaults)
+
+	// Telemetry
+	telemetryDefaults := telemetryv1.TelemetryDefaults{
+		CentralContainerImageURL:      os.Getenv("CEILOMETER_CENTRAL_IMAGE_URL_DEFAULT"),
+		CentralInitContainerImageURL:  os.Getenv("CEILOMETER_CENTRAL_INIT_IMAGE_URL_DEFAULT"),
+		ComputeContainerImageURL:      os.Getenv("CEILOMETER_COMPUTE_IMAGE_URL_DEFAULT"),
+		ComputeInitContainerImageURL:  os.Getenv("CEILOMETER_COMPUTE_INIT_IMAGE_URL_DEFAULT"),
+		NotificationContainerImageURL: os.Getenv("CEILOMETER_NOTIFICATION_IMAGE_URL_DEFAULT"),
+		SgCoreContainerImageURL:       os.Getenv("CEILOMETER_SGCORE_INIT_IMAGE_URL_DEFAULT"),
```

Comment:
Bad copy-paste here of a mistake I made in the initial Telemetry operator webhook PR.  Need to fix this: `s/CEILOMETER_SGCORE_INIT_IMAGE_URL_DEFAULT/CEILOMETER_SGCORE_IMAGE_URL_DEFAULT`